src/ripple/app/ledger/impl/LedgerConsensusImp.cpp

```diff
@@ -1016,6 +1022,12 @@ void LedgerConsensusImp::accept (std::shared_ptr<SHAMap> set)
 
         buildLCL->updateSkipList ();
 
+        if (buildLCL->info().seq > v2_ledger_seq_switch &&
+            !buildLCL->stateMap().is_v2())
```

Comment:
It seems like this condition can never happen, because line 988 would have converted to version 2